cpp/src/cider/tests/nextgen/functional/CastTest.cpp

```diff
@@ -42,58 +44,56 @@
     }                                                                                    \
   };
 
-#define TEST_UNIT_FOR_ARROW(TEST_CLASS, UNIT_NAME)                                       \
-  TEST_F(TEST_CLASS, UNIT_NAME) {                                                        \
-    assertQueryArrow("SELECT CAST(col_a as TINYINT), CAST(col_b as TINYINT) FROM test"); \
-    assertQueryArrow(                                                                    \
-        "SELECT CAST(col_a as SMALLINT), CAST(col_b as SMALLINT) FROM test");            \
-    assertQueryArrow("SELECT CAST(col_a as INTEGER), CAST(col_b as INTEGER) FROM test"); \
-    assertQueryArrow("SELECT CAST(col_a as BIGINT), CAST(col_b as BIGINT) FROM test");   \
-    assertQueryArrow("SELECT CAST(col_a as FLOAT), CAST(col_b as FLOAT) FROM test");     \
-    assertQueryArrow("SELECT CAST(col_a as DOUBLE), CAST(col_b as DOUBLE) FROM test");   \
-    assertQueryArrow(                                                                    \
-        "SELECT CAST(col_a as DOUBLE)  FROM test where CAST(col_b as INTEGER) > 20 ");   \
-    assertQueryArrow(                                                                    \
-        "SELECT CAST(col_a as INTEGER) + CAST(col_b as INTEGER) FROM test");             \
-    GTEST_SKIP() << "Test skipped since groupby not ready";                              \
-    assertQueryArrowIgnoreOrder(                                                         \
-        "SELECT CAST(col_a as INTEGER), count(col_b) FROM test GROUP BY col_a", "");     \
+#define TEST_UNIT(TEST_CLASS, UNIT_NAME)                                               \
+  TEST_F(TEST_CLASS, UNIT_NAME) {                                                      \
+    assertQuery("SELECT CAST(col_a as TINYINT), CAST(col_b as TINYINT) FROM test");    \
+    assertQuery("SELECT CAST(col_a as SMALLINT), CAST(col_b as SMALLINT) FROM test");  \
+    assertQuery("SELECT CAST(col_a as INTEGER), CAST(col_b as INTEGER) FROM test");    \
+    assertQuery("SELECT CAST(col_a as BIGINT), CAST(col_b as BIGINT) FROM test");      \
+    assertQuery("SELECT CAST(col_a as FLOAT), CAST(col_b as FLOAT) FROM test");        \
+    assertQuery("SELECT CAST(col_a as DOUBLE), CAST(col_b as DOUBLE) FROM test");      \
+    assertQuery(                                                                       \
+        "SELECT CAST(col_a as DOUBLE)  FROM test where CAST(col_b as INTEGER) > 20 "); \
+    assertQuery("SELECT CAST(col_a as INTEGER) + CAST(col_b as INTEGER) FROM test");   \
+    GTEST_SKIP() << "Test skipped since groupby not ready";                            \
+    assertQueryIgnoreOrder(                                                            \
+        "SELECT CAST(col_a as INTEGER), count(col_b) FROM test GROUP BY col_a", "");   \
   }
 
-GEN_CAST_TYPE_TEST_CLASS_FOR_ARROW(Float, Fp32)
+GEN_CAST_TYPE_TEST_CLASS(Float, Fp32)
```

Comment:
@kd97 can you help review these update change?

Reply:
better to rebase code? seems no new change in your latest commit? @harborn